e2e/parent-arg-from-func-call/src/Preg.php

```diff
@@ -5,7 +5,7 @@ class Preg
     /**
      * @param string $subject
      */
-    public static function replaceCallbackArray(array $pattern, $subject, int $limit = -1, int &$count = null, int $flags = 0): string
+    public static function replaceCallbackArray(array $pattern, $subject, int $limit = -1, ?int &$count = null, int $flags = 0): string
```

Comment:
please revert this, this is e2e test, changing param test for test can change scope that was gone on purpose to be fixed in iteration for scoping check filling effort, ref

- https://github.com/rectorphp/rector-src/pull/2492
- https://github.com/rectorphp/rector-src/pull/2493
- https://github.com/rectorphp/rector-src/pull/2495
- https://github.com/rectorphp/rector-src/pull/2496
- https://github.com/rectorphp/rector-src/pull/2497